convert_client.c

```diff
@@ -650,29 +558,28 @@ _validate_parameters(char *err_buf, size_t len)
 	}
 
 	/* resolve address */
-	if (_validate_and_set_convert_addr(convert_addr) < 0) {
+	if (_validate_and_set_convert_addr(convert_addr) != 0) {
```

Comment:
can be replaced by `getaddrinfo(convert_addr, _convert_port, NULL, &_converter_addr)` directly

Reply:
Just fixed that, thanks